packages/fxa-auth-server/lib/routes/account.ts

```diff
@@ -590,12 +590,24 @@ export class AccountHandler {
         }
       );
       await this.db.resetAccountTokens(uid);
+      const sessionToken = await this.createSessionToken({
+        account,
+        request,
+        // this route counts as verification
+        tokenVerificationCode: undefined,
+        tokenVerificationId: undefined,
+      });
+      return {
+        uid,
+        sessionToken: sessionToken.data,
+        verified: sessionToken.emailVerified,
```

Comment:
Would the email on the sessionToken ever be unverified? Could we return true here?

Reply:
it should always be true, but i think it's more helpful to reference the source of the value.